packages/main/src/themes/Icon.css

```diff
@@ -28,8 +28,17 @@
 }
 
 .ui5-icon-root {
+    width: 1rem;
```

Comment:
I think this change will break all icon scaling:
![image](https://user-images.githubusercontent.com/15844574/86348258-19946e00-bc68-11ea-8c88-f1ad8b3f509a.png)
this f.e. is set to 50 by 50px

Should become:
![image](https://user-images.githubusercontent.com/15844574/86348317-303ac500-bc68-11ea-87a5-dd2f156f2895.png)